.github/workflows/artifact-links.yml

```diff
@@ -0,0 +1,20 @@
+name: Artifact links comments creator
+on:
+  workflow_run:
+    workflows: ["PR"]
+    types: [completed]
+
+jobs:
+  artifacts-url-comments:
+    name: Add artifact links to PR and issues
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Add artifact links to PR and issues
+        uses: tonyhallett/artifacts-url-comments@v1.1.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          prefix: "Do you want to test this code? Here you have an automated build:"
+          suffix: "WARNING: It may be unstable. Use only for testing!"
```

Comment:
Install the unified hex is not straightforward, what about adding a link here to a video with instructions? For example this from mikeller: https://www.youtube.com/watch?v=I1uN9CN30gw but it will serve any other if we find other easier for users.

Reply:
Do we have a document we can link to that embeds the video?